bakery/checkers/operations.go

```diff
@@ -14,6 +14,8 @@ type opKey struct{}
 // given operations. An allow caveat will succeed only if one of the allowed
 // operations is in ops; a deny caveat will succeed only if none of the denied
 // operations are in ops.
+//
+// This uses the same context key as ContextWithOpAuthorizer.
```

Comment:
What's that then, and why are there two functions that do the same thing?

Reply:
Ah, missed that - it's from the next PR. Ignore this comment - all should become clear in time.